open_instruct/dpo_tune.py

```diff
@@ -209,6 +213,21 @@ def main(args: FlatArguments):
     # Initialize the accelerator. We will let the accelerator handle device placement for us in this example.
     # If we're using tracking, we also need to initialize it here and it will by default pick up all supported trackers
     # in the environment
+    exp_name = os.path.basename(__file__)[: -len(".py")]
+    if args.push_to_hub:
+        if args.hf_repo_id is None:  # auto-generate one
+            args.hf_repo_id = "open_instruct_dev"
+        if args.hf_entity is None:  # first try to use AI2 entity
+            args.hf_entity = maybe_use_ai2_hf_entity()
+        if args.hf_entity is None:  # then try to use the user's entity
+            args.hf_entity = HfApi().whoami()["name"]
+        args.hf_repo_id = f"{args.hf_entity}/{args.hf_repo_id}"
+        if args.hf_repo_revision is None:  # auto-generate one
```

Comment:
so is the intended use to upload multiple runs to the same 'model' location but under different revisions? or each finetune run as a separate model?

Reply:
Yes, something like this. 

<img width="887" alt="image" src="https://github.com/user-attachments/assets/dade3dd4-80e7-41ca-8ecd-4d186650e32d">

We should mainly use the wandb to find models. The HF model stuff is just a dump (I suppose redundancy).

<img width="591" alt="image" src="https://github.com/user-attachments/assets/a822d770-0ab1-43e3-a1c8-a6ac7c0a9d72">